### PR TITLE
FormatNumber fails for Chicago style with roman numbers

### PR DESCRIPTION
**Pull request by bmoelans**
The problem can be simulated as

```
import re

first = 'I'  # or another roman number
m = re.search(r'\d+', first)
first_number = int(m.group())
```

In this case `m is None`

### CODE REVIEW

**Review by tmorrell — APPROVED** [review]
Thanks for this contribution! I don't see any test cases that match this use case....but returning something if the regex doesn't match seems reasonable. I'll merge shortly unless someone comes up with a reason not to in the next few days.

**Review by yarikoptic — APPROVED** [review]
Sure thing proposed change would make code to not break.

But if code to support roman letters here, shouldn't they be just converted to numeric? This fix would just totally ignore non-numeric "first" and return "last" as is ... but this whole "chicago" style for page numbers is quite an abomination, so I would not sweat over, so let's just make it happen after clarifying on that `[0:]`

File: citeproc/model.py [hunk 1]
```diff
@@ -666,6 +666,8 @@ def find_common(first, last):
         common = find_common(first, last)
         if range_format == 'chicago':
             m = re.search(r'\d+', first)
+            if not m:
+                return last[0:]
```
[the remark this answers is not in the document]
Comment (bmoelans) [reply citeproc/model.py]: using `last[0:]` instead of `last` is indeed stupid, if fixed this

**Comment by bmoelans** [comment]
Thx for merging. Will there come soon a new release so I can just do `pip install citeproc-py --update` ?

**Pull request merged**